systemd/ceph-disk@.service

```diff
@@ -1,11 +0,0 @@
-[Unit]
```

Comment:
Just to make sure: Are there no other dependencies on ceph-disk in the other systemd unit files?

Reply:
There aren't. At least, I couldn't find any. ceph-disk units would trigger ceph-osd units usually, I don't know of anything the other way around.

Reply:
I thought ceph-osd had something like After=ceph-disk line but I might have been wrong. Looking at the master branch, there does not seem to be anything like that.